.github/workflows/functional-test.yaml

```diff
@@ -296,7 +296,7 @@ jobs:
       fail-fast: true
       matrix:
         os: [ubuntu-latest]
-        name: [ucp,shared,msgrp,daprrp,samples]
+        name: [ucp,kubernetes,shared,msgrp,daprrp,samples]
```

Comment:
Adding a new run to the matrix, this doesn't really belong in the other categories.